src/client/src/rt-components/loadable/Loadable.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { Component } from 'react'
+import { styled } from 'rt-theme'
+import { ServiceConnectionStatus } from 'rt-types'
+import DisconnectIcon from '../icons/DisconnectIcon'
+
+const LoadableStyle = styled.div`
+  width: 100%;
+  height: 100%;
+  border-radius: 0.1875rem;
+  background-color: ${({ theme }) => theme.component.backgroundColor};
+  color: ${({ theme }) => theme.component.textColor};
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+  opacity: 0.59;
+  fill: ${({ theme }) => theme.component.textColor};
+`
+
+const Spinner = styled.div`
+  margin: 6.25rem auto;
+  width: 3.125rem;
+  height: 2.5rem;
+  text-align: center;
+  font-size: 0.625rem;
+`
+
+const Rect = styled.div<{ delay?: number }>`
+  background-color: ${({ theme }) => theme.component.textColor};
+  margin: 0 0.0625rem;
+  height: 100%;
+  width: 0.375rem;
+  display: inline-block;
+
+  animation: stretchdelay 1.2s infinite ease-in-out;
+
+  animation-delay: -${({ delay }) => delay}s;
+
+  @keyframes stretchdelay {
```

Comment:
Using the `styled` keyframes tool avoid name collisions — also, it's kinda cool 😛 
https://emotion.sh/docs/keyframes